include/superior_mysqlpp/uncaught_exception_counter.hpp

```diff
@@ -4,32 +4,19 @@
 
 #pragma once
 
-#include <cxxabi.h>
+#include <exception>
 
 namespace SuperiorMySqlpp
 {
-    inline int uncaughtExceptions() noexcept
```

Comment:
I didn't realize before this is not a detail/private, so this removal is a breaking change, we must be aware of that, perhaps even keep this method as deprecated for some time. Opinions, @smradCZ ?

Reply:
I removed it because you mentioned that this implementation it's some kind of hack.

I provided you a solution that worked just fine, but you wanted to use something more portable (e.g. `std::current_exception` instead of some function from `abi`). so if you want me to keep this method i will revert changes of `UncaughtExceptionCounter`.